lib/presentation/widget/myspace/my_space_widget.dart

```diff
@@ -51,6 +52,7 @@ class MySpaceWidget extends StatefulWidget {
 }
 
 class _MySpaceWidgetState extends State<MySpaceWidget> {
+  final _scaffoldKey = GlobalKey<ScaffoldState>();
```

Comment:
what is using for?

Reply:
I can't target the scaffold with its context because it's in a builder so I need to tag it with a key. I'll try to find an other way later

Reply:
oh no, this is the question for me to understand. I am going to use key in widget for UI testing in the future too. So I want to understand that. Please investigate on that, and distinguish `ValueKey`, `GlobalKey`, `UniqueKey`,... what is the good for us: 
- not only to target the widget like you
- but also for `automation test` can find exactly widget with `key`